### DISCUSSION TITLE
What settings were used to get "5GB of size in just 85 seconds"?

### DISCUSSION THREAD

**Discussion by TheMrGong** [forscht/ddrive#66] [General]
I'm curious how many webhooks and what concurrency was set to to achieve that - and whether it's actually a good idea to do it

**Comment by forscht** [comment]
@TheMrGong Sorry for the late reply, Somehow I am not getting notifications for discussion. I tested it when chunksize was 8MB so settings might deffer now.

**Test settings**
1. Webhooks - 10
2. Concurrency - 20

**Result with 1GB**
![image](https://github.com/forscht/ddrive/assets/59018146/398451f8-de52-432e-87f0-339e8b8ccfb7)

**Result with 5GB**
![image](https://github.com/forscht/ddrive/assets/59018146/269e67a2-2523-4113-89ac-0b5231fee0fd)
